crt_portal/cts_forms/templates/forms/errors.html

```diff
@@ -27,9 +27,9 @@
       <div class="crt-error-card">
         <div class="crt-error-card__content">
           <div class="display-flex flex-column padding-bottom-3">
-            <h1 class=".h1__display">{{ status }}</h1>
-            <span>{{ message }}</span>
-            <span class="help-text__small">{{ helptext }}</span>
+            <h1 class=".h1__display">{% block error_heading %}We're sorry, something went wrong{% endblock %}</h1>
+            <span class="crt-error_message">{% block error_message %}We hope to fix this issue shortly{% endblock %}</span>
+            <span class="help-text__small">{% block error_helptext %}Error: {{ status }}{% endblock %}</span>
```

Comment:
Did we change the order from `status`/`message`/`helptext`? it's a bit confusing here since the classes don't reflect this change. 

Side note, using blocks seems overly complicated for this use case -- usually we want blocks to override more complicated functionality. Here, I believe we can do something simpler: `{{ message|default:"We're sorry, something went wrong" }}` instead.